### ISSUE TITLE
Icons are missing hover text for screen readers

### ISSUE THREAD

**Issue by KTrentLR** [zenoamaro/react-quill#624]
There should be default hover text for each button/icon for screen readers. There should be a way to set custom hover text for each icon (to support multiple languages).

**Comment by AbhisheshPradhan** [comment]
Yes, it is missing the hover text on the toolbar buttons. For this, I simply created a custom toolbar and added **title** attribute.

`const CustomToolbar = () => (
<div id='toolbar'>
<select className='ql-header' title='Size' defaultValue={''} onChange={(e) => e.persist()}>
  <option value='1' />
  <option value='2' />
  <option selected />
</select>
<button className='ql-bold' title='Bold' />
<button className='ql-italic' title='Italic' />
<select className='ql-color' title='Color'>
  <option value='red' />
  <option value='green' />
  <option value='blue' />
  <option selected />
</select>
</div>
);

const EditorModule = {
toolbar: {
container: '#toolbar',
},
clipboard: {
matchVisual: false,
},
};


  <>
                    <CustomToolbar />
                    <ReactQuill
                      value={this.state.value}
                      onChange={this._onChange}
                      modules={EditorModule}
                      theme={'snow'} // pass false to use minimal theme
                    />
                  </>
`

**Comment by alexkrolick** [comment]
This should be raised upstream in Quill, since the DOM is created by Quill, not the wrapper
